packages/flutter_tools/lib/src/base/user_messages.dart

```diff
@@ -66,15 +66,15 @@ class UserMessages {
       'Unable to locate Android SDK.\n'
       'Install Android Studio from: https://developer.android.com/studio/index.html\n'
       'On first launch it will assist you in installing the Android SDK components.\n'
-      '(or visit https://flutter.dev/setup/#android-setup for detailed instructions).\n'
+      '(or visit https://flutter.dev/docs/get-started/install#android-setup for detailed instructions).\n'
```

Comment:
that link still loses the `#android-setup` by the time you select your platform.

Reply:
it won't be https://flutter.dev/docs/get-started/install anymore, I think it's still better then https://flutter.dev/setup/ that is redirect only

Reply:
Would https://flutter.dev/docs/get-started/install without the anchor be better (since it doesn't imply any android info), plus create an issue to switch on https://flutter.dev/docs/get-started/install/windows, https://flutter.dev/docs/get-started/install/macos etc based on the current platform?

Reply:
Or you can make that platform switch happen in this review if you want, @Dmitry-Borodin

Reply:
Will check tomorrow how can easy can I get the platform in dart.

Reply:
@jmagman updated PR, made urls platform dependent, please take a look
I tested some random messages on my computer, but don't see a nice possibility how to add autotests to the project.
The code is straightforward and can run on any platform, so test will just copy the code it should test